exonum/Cargo.toml

```diff
@@ -51,11 +51,13 @@ chrono = { version = "=0.4.6", features = ["serde"] }
 uuid = { version = "=0.7.1", features = ["serde"] }
 snow = "=0.4.0"
 rust_decimal = "=0.10.2"
-protobuf = "=2.2.0"
+protobuf = { version = "=2.2.0", features = ["with-serde"] }
+ring = "=0.13.2"
```

Comment:
@dvush As we deleted travis job with build 1.27.2 version of Rust so this dependency not needed anymore.